docs/book/v3/standard-filters.md

```diff
@@ -861,7 +861,7 @@ It is best practice is to use the `TYPE_*` constants rather than the human-reada
 
 ## ToString
 
-The `ToString` filter casts `Stringable` objects or scalar values to `string`.
+The `ToString` filter casts `Stringable` objects or scalar values to `string`. Iterable values will be casted recursive
```

Comment:
Unsure if it make sense to make this configurable with an option if the user want to use it or not

Reply:
I don't think an option is necessary as long as the documented behaviour is clear